src/p5.dance.interpreted.js

```diff
@@ -123,7 +125,9 @@ function runUserEvents(events) {
   }
 
   function executeFuncs(item) {
+    setFuncContext(item.eventType, item.param);
```

Comment:
Would `setInputContext` be a clearer name for this?

Reply:
After Slack discussion, we agreed to keep `setFuncContext`. Thanks!